src/pkg/mcp/tools/common.go

```diff
@@ -26,3 +29,13 @@ func configureLoader(request mcp.CallToolRequest) *compose.Loader {
 
 	return compose.NewLoader()
 }
+
+// HandleTermsOfServiceError checks if the error is related to terms of service not being accepted
+// and returns an appropriate error message if it is.
+// Returns nil if the error is not related to terms of service.
+func HandleTermsOfServiceError(err error) *mcp.CallToolResult {
+	if connect.CodeOf(err) == connect.CodeFailedPrecondition && strings.Contains(err.Error(), "terms of service") {
+		return mcp.NewToolResultError("The deployment failed because the terms of service were not accepted. Please accept the terms of service by logging in here: https://portal.defang.io/auth/login. Then try again.")
```

Comment:
Should we include the original `err`? Just in case there's something contextual, like "new version of TOS".